custom_components/adjustable_bed/beds/okin_nordic.py

```
@@ -0,0 +1,379 @@
+            pulse_count = self._coordinator.motor_pulse_count
+            pulse_delay = self._coordinator.motor_pulse_delay_ms
+            await self.write_command(
+                command, repeat_count=pulse_count, repeat_delay_ms=pulse_delay
+            )
+        finally:
+            try:
+                await self.write_command(
+                    OkinNordicCommands.STOP,
+                    cancel_event=asyncio.Event(),
+                )
+            except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)